notebooks/analog_training_LeNet5.ipynb

```diff
@@ -118,15 +117,27 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "colab": {
      "base_uri": "https://localhost:8080/"
     },
     "id": "8-uJPmFMzHeX",
     "outputId": "d9c997cf-cd3c-47ab-c005-e474d1f98fd6"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "tensor([[-0.1216,  0.4219],\n",
+       "        [-0.2063,  0.5724]], grad_fn=<AddBackward0>)"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
@kkvtran can you delete your hidden data of the notebook (clear cell outputs)

Reply:
OK, I can do the kernel restart and remove all the existing output.